src/index.yaml

```diff
@@ -0,0 +1,196 @@
+layout: layouts/landing.jsx
+title: probe-rs - the embedded toolkit
+description: Support for multiple file formats like Markdown, YAML, JavaScript, TypeScript, JSX, Nunjucks etc.
+header:
+  title: The <span>user-friendly & flexible</span> embedded toolkit <span> that works </span>
+
+install:
+  title: "Run the following or follow the [instructions](./docs/overview/installation.md):"
+  code: "curl -LsSf https://github.com/probe-rs/probe-rs/releases/download/0.20.0/probe-rs-installer.sh | sh"
```

Comment:
Could we use a a link which doesn't depend on the version number here?

Also, this is cut of on my screen:

![image](https://github.com/probe-rs/webpage/assets/859555/577d8765-a351-4637-a67c-8e33a0b72fc0)

Reply:
So the cutoff is not really fixable because this ofc depends on the screen size. Scrolling helps tho. We could do some shenanigans with URL forwarding maybe that shortens the URL and removes the version. But it's a bit more work. For now I'd like to keep it this way. I adjusted everything to include the current release version.